gitlab-runners/ubi8-git/version.json

```diff
@@ -0,0 +1 @@
+{"version":"v0.1.0"}
```

Comment:
why not just make this v1.0? seems silly to never reach v1.0 in my mind.

Reply:
As the original contributor of this runner I was going to ask you about what version you wanted anyway :grinning: 
Silly or not, the idea was that you as the contributor of the "thing" would be in control of this anyway and thus can create new tags/releases at your own discretion. This PR is just to get the ball rolling on pushing images to Quay.